<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrenchPointsElevationDialogBase</class>
 <widget class="QDialog" name="FrenchPointsElevationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>French Points Elevation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="qt_text_intro">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="qt_label_layer">
     <property name="text">
      <string>Couche source</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsMapLayerComboBox" name="qt_cb_selectedLayer"/>
   </item>
   <item>
    <widget class="QLabel" name="qt_message_selectedLayerInfos">
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="qt_label_field">
     <property name="text">
      <string>Nom du champ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="qt_fieldName">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>z</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="qt_message_fieldName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="qt_label_resource">
     <property name="text">
      <string>Ressource</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="qt_cb_selectedResource"/>
   </item>
   <item>
    <widget class="QLabel" name="qt_message_resource">
     <property name="text">
      <string/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="qt_progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="qt_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qt_button_box</sender>
   <signal>rejected()</signal>
   <receiver>FrenchPointsElevationDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
